src/TranslatorSettingsDialog.ui

```diff
@@ -6,63 +6,155 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>300</width>
-    <height>244</height>
+    <width>362</width>
```

Comment:
Every time one of us works on QtCreator, we have phantom resizes here and there. Not sure how we can completely avoid that.

Reply:
Probably by being a bit more strict in which lines we commit. This time it was intentional though, 300px was a bit too small for the dialog to also contain the tab pane and the table. 362 is a bit random…

Reply:
Not a criticism, just an observation. I've noticed this with my commits as well.